### ISSUE TITLE
treesitter support

### ISSUE THREAD

**Issue by exoess** [lunacookies/vim-colors-xcode#37]
could you add builtin treesitter support?

**Comment by bronzehedwick** [comment]
Yes, I think this is good to support explicitly. Lots of treesitter groups are already linked, so you'll get a decent experience out of the box, but more support would be nice. I'll try to spend some time thinking on this soon.
